targets/targets.json

```diff
@@ -6644,6 +6645,7 @@
             "PORTINOUT",
             "PORTOUT",
             "PWMOUT",
+            "RTC",
```

Comment:
Are you sure these platforms actually support RTC as defined by Mbed? I remember RTC in some Nordic targets couldn't survive reset.

Reply:
Good question! @desmond-blue  - Can you comment?  Not sure why we have modified that Delta NRF51 target and not sure why we need the RTC for the NRF52840_DK.  Master doesn't have it defined and in sdk_config.h we are still not enabling RTC.

Reply:
Chatted with @desmond-blue.  This is a test left-over by me... I've corrected it with cfd4cb260ec284de45855886f1ddc61d9a841e0e.